.github/CODEOWNERS

```diff
@@ -23,13 +23,13 @@ conftest.py @lbianchi-lbl
 /idaes/commands/ @dangunter
 
 # Core
-/idaes/core/ @andrewlee94 @dallan-keylogic @lbianchi-lbl
+/idaes/core/ @agarciadiego @dallan-keylogic @lbianchi-lbl
 /idaes/core/dmf/ @dangunter
-/idaes/core/surrogate/ @andrewlee94 @bpaul4 @avdudchenko @rundxdi
+/idaes/core/surrogate/ @bpaul4 @avdudchenko @rundxdi
 /idaes/core/ui/ @dangunter
 
 # Models
-/idaes/models/ @andrewlee94 @bpaul4
+/idaes/models/ @agarciadiego @bpaul4
```

Comment:
Add me to `/idaes/models/property_models/`

Reply:
Done.